### ISSUE TITLE
FEATURE - More control over training/validation set during experiment

### ISSUE THREAD

**Issue by KaikeWesleyReis** [autonomio/talos#445]
#### 1) I think Talos should add this feature:
Implement some more parameters for the Scan function:
- ```x_val```: set different from x, to use as validation only!
- ```y_val```:set different from y, to use as validation only!
- ```validation_split```: if ```x_val``` / ```y_val``` is ```None```, define x split to validate your model

The idea here is to allow the user more control over the experiment, allowing to choose a database to use as validation or the portion of ```x``` / ```y``` to use as validation. These options are equivalent to the options in `` `keras```.

#### 2) Why?

In models development it's necessary to have a higher control over the train/validation set to ensure the reproducibility of the experiment.

**For example**: let's say I created a model A with certain hyper parameters and considering defined train/validation set (keras CIFAR-10 is a good example). However, I would like to evaluate other hyper parameters for my architecture using *talos*. To do this, it's necessary to fix, as model A, the original train/validation set for my experiment to give the opportunity to compare and to ensure the reproducibility of the code on other machines.

#### 3) I believe this feature is

- [ ] critically important
- [X] must have
- [ ] nice to have

#### 4) Given the chance, I'd be happy to make a PR for this feature
- [ ] definitely
- [X] possibly
- [ ] unlikely

But to a PR, I need to understand how/where *talos* does train validation split, if you show me I would glad help this package improvement (it is awesome).

**Comment by mikkokotila** [comment]
`Scan()` already supports this. Have a look at https://autonomio.github.io/talos/#/Scan. 

Closing this as resolved. Feel free to open new issue if anything.
